src/broker/WebSocketShim.cc

```diff
@@ -176,6 +179,9 @@ void WebSocketShim::ProcessMessage(std::string_view topic, broker::zeek::Event&
         return;
     }
 
+    size_t size = ev.as_data().shared_envelope()->raw_bytes().second;
```

Comment:
Could you update `shared_envelope` so it returns a `const auto&`? Right now just getting this count causes an atomic increment and decrement.

Reply:
Opened PR: https://github.com/zeek/broker/pull/470